exporter/metric/prometheus/prometheus.go

```diff
@@ -0,0 +1,384 @@
+// Copyright 2019, OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package prometheus
+
+import (
+	"context"
+	"go.opentelemetry.io/otel/sdk/metric"
+	"net/http"
+	"sync"
+
+	"github.com/prometheus/client_golang/prometheus/promhttp"
+
```

Comment:
import formatting.

Reply:
I've grouped them in two blocks: stdlib and others. From what I could see, this is more common in this repo but this is not very uniform on the whole project. 

I think we need to define how we should format imports and validate this in the CI.